docs/source-app/workflows/access_app_state/access_app_state.rst

```diff
@@ -0,0 +1,61 @@
+.. _access_app_state:
+
+################
+Access App State
```

Comment:
It could be confusing to have a page named like this. We also have a class AppState which is completely separate from this topic.  cc @tchaton

Reply:
Updated

Reply:
@awaelchli @tchaton 

How about it gets renamed to something like: Communication in Lightning Apps (which is what a Google search returns) , and I do some rewrites around the topic?

Reply:
That would be great! I'm fine either way though, just wanted to mention it.

Reply:
I believe it is done, if not let's address it in additional PR...